templates/list.html

```diff
@@ -28,7 +29,21 @@ <h1 class="text-xl font-bold">{{ list.title }}</h1>
       hx-select="#create_link"
       >Link an item</a
     >
+    <a
```

Comment:
After clicking this link, the whole section is empty. You'll need to add `id="toggle_publicity"`, remove the `hx-target` attribute and add `hx-swap="outerHTML"` to make it work.